Gems/Terrain/Assets/Shaders/Terrain/ClipmapComputeHelpers.azsli

```diff
@@ -229,10 +241,15 @@ BilinearUvs CalculateClipmapUv(
     return uvs;
 }
 
-float4 ColorPointSampling(Texture2DArray<float4> colorClipmap, uint2 uv, uint clipmapLevel)
+float4 ColorPointSampling(Texture2DArray<float4> colorClipmap, uint2 uv, uint clipmapLevel, float debugEnabled)
```

Comment:
float debugEnabled is slightly confusing
float debugFactor
or
bool debugEnabled
would be clearer.